### ISSUE TITLE
Building docs produces lots of warnings

### ISSUE THREAD

**Issue by akrabat** [Redocly/redoc#2549]
**Describe the bug**

 Building docs using the Docker image for redocly/cli produces more than a screenful of warning messages. 

**To Reproduce**
Steps to reproduce the behaviour:

1. Download the Museum example API: https://raw.githubusercontent.com/Redocly/museum-openapi-example/main/openapi.yaml and place in `doc/`
2. Build the docs using the Docker container: `docker run --rm -v $PWD/doc:/spec redocly/cli build-docs openapi.yaml`
3. See the output:

![Screenshot 2024-06-09 at 15 29 01](https://github.com/Redocly/redocly-cli/assets/33135/52e0036b-baa7-4921-930f-3f28101b5b49)

Full output at https://gist.github.com/akrabat/5a75b89104a884c1128d4f251078628b

**Expected behavior**

Successfully building the docs should show no warnings related to the build environment and only warnings related to my OpenAPI spec (if any).

**Redocly Version(s)**

```
$ docker run --rm -v $PWD/doc:/spec redocly/cli --version
1.14.0
```

**`Node.js` Version(s)**

Whatever is in the Docker container

**Comment by AlexVarchuk** [comment]
fixed via #2547

@tatomyr please upgrade redoc to `v2.1.5` in `@redocly/cli`

**Comment by tatomyr** [comment]
Thanks @AlexVarchuk!
